src/pages/productPage.ts

```diff
@@ -103,4 +111,14 @@ export default class ProductPage extends BasePage {
       await expect(this.productPageLocators.productCountInCompareListText).toHaveText(expectedRegex, { timeout: 15000 });
     });
   }
+
+  async verifyProductInWishlistByName(productName: string) {
+    await test.step(`Verify product "${productName}" is added to Wishlist`, async () => {
+      // Локатор для повідомлення про успішне додавання до списку бажаного
+      const successMessageLocator = this.page.locator('div.message-success.success.message');
```

Comment:
Define locator elements in appropriate page class section. Add locator name to locators list and initialize it in constructor function